pkg/config/utils.go

```diff
@@ -146,7 +147,7 @@ func FindAllStackConfigsInPaths(
 		if len(allMatches) == 0 {
 			_, err := u.GetGlobMatches(patterns[0])
 			if err != nil {
-				if atmosConfig.Logs.Level == u.LogLevelTrace {
+				if atmosConfig.Logs.Level == u.LogLevelDebug {
```

Comment:
Why are we using log levels outside of logging? Think of this like in IaC code doing: "if env = prod", we should never do that; instead, we should feature flag the behavior, not the environment. The log level is like an environment, and it's use is only for logging. Not for controlling the flow of the atmos CLI outside of logging.

Reply:
Why don't we call `log.Debug` on `y`, and avoid returning an error with the entire config as a dynamic error? It's a double whammy.